source/plugins/wakatime/index.mjs

```diff
@@ -1,3 +1,22 @@
+async function pickOnlyGithubPublicRepos ({projects, axios, login, limit}) {
+  const result = []
+
+  for await (const project of projects) {
+    if (result.length >= limit) break
+    try {
+      console.debug(`metrics/compute/${login}/plugins > wakatime > checking 'https://github.com/${login}/${project.name}'`)
+      await axios.head(`https://github.com/${login}/${project.name}`)
+
+      result.push(project)
+    }
+    catch {
+      continue
+    }
+  }
+
+  return result
+}
+
```

Comment:
Can you move this function below the "main" function just so the actual plugin logic appears first when reading the file ?